cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html

```diff
@@ -46,7 +46,7 @@
 
 {% block content_main %}
     <h1>Bureau Structure</h1>
-    <p>Org chart last updated: May 3, 2016</p>
+    <p>Org chart last updated: August 3, 2016</p>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/47806847c47c1c2cf6b344c23e4223df2fe355c5